dev/tests/api-functional/testsuite/Magento/GraphQl/Wishlist/WishlistTest.php

```diff
@@ -0,0 +1,87 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+declare(strict_types=1);
+
+namespace Magento\GraphQl\Wishlist;
+
+use Magento\Integration\Api\CustomerTokenServiceInterface;
+use Magento\TestFramework\TestCase\GraphQlAbstract;
+use Magento\Wishlist\Model\Item;
+
+class WishlistTest extends GraphQlAbstract
```

Comment:
Please cover adding items to the wishlist with tests.

Reply:
yes. Thanks. I will